pkg/api/v1/status/condition.go

```diff
@@ -66,7 +66,22 @@ func TrueCondition(conditionType ConditionType) Condition {
 		Type:               conditionType,
 		Status:             corev1.ConditionTrue,
 		LastTransitionTime: metav1.Now(),
-		Reason:             "",
-		Message:            "",
 	}
 }
+
+// FalseCondition returns the COndition that has the 'Status' set to 'false' and 'Type' to 'conditionType'.
```

Comment:
[nit] typo in **CO**ndition

Reply:
will fix in the current PR that is based on the current one